optimize/util/optimize-commons/src/test/resources/environment-variable-default-value-test-config.yaml

```diff
@@ -11,27 +11,9 @@ api:
 
 engines:
   'camunda-bpm-1':
```

Comment:
I feel like this whole block _should_ be removable, but I would like to learn why if this is not the case